packages/core/src/components/forms/_file-upload.scss

```diff
@@ -95,17 +81,9 @@ $file-upload-button-width-large: $pt-grid-size * 8.5;
     content: "Browse";
   }
 
-  &:hover::after {
-    @include pt-button-hover();
-  }
-
-  &:active::after {
-    @include pt-button-active();
-  }
-
   .pt-large & {
```

Comment:
❌ this should be `&.pt-large`, but the contents apply to `.pt-file-upload-input` (which wants `.pt-large &`). 
needs more refactoring.